soccer/Robot.cpp

```diff
@@ -153,8 +152,10 @@ void OurRobot::resetForNextIteration() {
 }
 
 void OurRobot::resetMotionConstraints() {
+    _rotationConstraints = RotationConstraints();
     _motionConstraints = MotionConstraints();
-    _motionCommand = Planning::MotionCommand();
+    _motionCommand = make_unique<Planning::EmptyCommand>();
```

Comment:
Should we handle the empty/none case the same for both angle and motion commands?  Maybe they should both be nullptr or both be EmptyCommand.